lib/tasks/evm_settings.rake

```diff
@@ -81,16 +81,22 @@ module EvmSettings
   private_class_method :keyval_to_hash
 
   def self.str_to_value(value)
-    return true  if value =~ /true/i
-    return false if value =~ /false/i
-    return nil   if value =~ /nil/
+    return value unless value.kind_of?(String)
+
+    dvalue = value.downcasex
```

Comment:
```suggestion
    dvalue = value.downcase
```

Reply:
why the heck did all these tests pass?

I need to add tests to this method. thanks